lib/browser/guest-window-manager.js

```diff
@@ -34,7 +34,7 @@ const mergeOptions = function (child, parent, visited) {
     if (key in child && key !== 'webPreferences') continue
 
     const value = parent[key]
-    if (typeof value === 'object') {
+    if (typeof value === 'object' && !Array.isArray(value)) {
```

Comment:
This was a bug I found relevant to this change.  `additionalArguments` is an array and wasn't being inherited properly